content/components/text-inputs/examples/example-width/example-code.md

```diff
@@ -0,0 +1,14 @@
+<div class="au-body example-form-item">
+  <label for="text-input">label text</label>
+  <input class="au-text-input au-text-input--width-sm" name="text-input-sm" id="text-input-sm" type="text" value="ABCZ abcjg liI1 12340">
```

Comment:
should we put less text in this text box, since it's smaller?

Reply:
good point, resolved in previous commit